### ISSUE TITLE
[Bug] error: unexpected argument '--beacon' found

### ISSUE THREAD

**Issue by lispking** [ProvableHQ/snarkOS#2992]
## 🐛 Bug Report

https://developer.aleo.org/testnet/getting_started/deploy_execute
<img width="509" alt="image" src="https://github.com/AleoHQ/snarkOS/assets/4446580/858f0d50-cca9-469d-a868-91e52c0ec450">

## Steps to Reproduce

$ snarkos start --nodisplay --dev 0 --beacon ""
error: unexpected argument '--beacon' found

Usage: snarkos start <--network <NETWORK>|--validator|--prover|--client|--private-key <PRIVATE_KEY>|--private-key-file <PRIVATE_KEY_FILE>|--node <NODE>|--bft <BFT>|--peers <PEERS>|--validators <VALIDATORS>|--rest <REST>|--norest|--nodisplay|--verbosity <VERBOSITY>|--logfile <LOGFILE>|--metrics|--cdn <CDN>|--nocdn|--dev <DEV>|--dev-num-validators <DEV_NUM_VALIDATORS>>

For more information, try '--help'.

## Your Environment

- snarkOS Version: 2.2.7
- Rust Version: 1.75.0
- Release: https://github.com/AleoHQ/snarkOS/releases/download/v2.2.7/aleo-testnet1-v2.2.7-x86_64-unknown-linux-gnu.zip

**Comment by ljedrz** [comment]
Nodes can no longer be started with a `--beacon` argument; the docs should be updated to reflect this.

**Comment by lispking** [comment]
Yes, the official documentation has not been updated, please take care of it, thank you.

**Comment by raychu86** [comment]
I believe this can be closed, as I don't see the `--beacon` flag in the documentation.

**Comment by cbeck88** [comment]
Closing as obsolete: the `--beacon` argument no longer exists in the CLI. The only remaining match in the tree is snarkVM's unrelated `vm.genesis_beacon()`, so the reported error can no longer occur.
